data/dev.tchx84.Portfolio.desktop.in

```diff
@@ -4,7 +4,8 @@ Name=Portfolio
 Exec=dev.tchx84.Portfolio
 Terminal=false
 Type=Application
-Categories=Utility;
+GenericName=File Manager
+Categories=GTK;System;Utility;FileTools;FileManager;
```

Comment:
We should probably stick to one main category :)

```
 desktop-file-validate /var/lib/flatpak/app/dev.tchx84.Portfolio/current/active/files/share/applications/dev.tchx84.Portfolio.desktop

/var/lib/flatpak/app/dev.tchx84.Portfolio/current/active/files/share/applications/dev.tchx84.Portfolio.desktop: hint: value "GTK;System;Utility;FileTools;FileManager;" for key "Categories" in group "Desktop Entry" contains more than one main category; application might appear more than once in the application menu
```

Reply:
Is FileManager the preferred category?

Reply:
Yes, it's exactly what it is :)

Reply:
This is what Nautilus has in its desktop file:
`Categories=GNOME;GTK;Utility;Core;FileManager;`

So, correct categories for Portfolio should be:
`Categories=GTK;Utility;FileManager;`

Also, consider adding some keywords as well.For example, based on Nautilus:
```
# TRANSLATORS: Search terms to find this application. Do NOT translate or localize the semicolons! The list MUST also end with a semicolon!
Keywords=folder;manager;explore;disk;filesystem;
```